.mega-linter.yml

```diff
@@ -13,6 +13,8 @@ DISABLE_LINTERS:
     - MARKDOWN_MARKDOWN_LINK_CHECK
     - PHP_BUILTIN
     - PHP_PSALM
+    # phpstan fails at the moment as it wants to run with PHP 8
```

Comment:
I don't understand this change, please explains ?

I've run locally with MegaLinter v5 (hash 7acb133f6), latest version available, as with GH Action [run](https://github.com/llaville/monolog-callbackfilterhandler/runs/7262050769?check_suite_focus=true#step:6:22)
and didn't need to disable PHPStan Linter 

```
----------------------------------------------------------------------------------------------------
-------------------------------------------- MegaLinter --------------------------------------------
----------------------------------------------------------------------------------------------------
 - Image Creation Date: 2022-06-26T14:30:08Z
 - Image Revision: 7acb133f6
 - Image Version: v5
```



```
### Processed [PHP] files
- Using [phpstan v1.7.15] https://megalinter.github.io/v5/descriptors/php_phpstan
- MegaLinter key: [PHP_PHPSTAN]
- Rules config: [.github/linters/phpstan.neon.dist]
✅ Linted [PHP] files with [phpstan] successfully - (2.6s)
[Text Reporter] Generated TEXT report: /tmp/lint/report/linters_logs/SUCCESS-PHP_PHPSTAN.log
```

Reply:
oh yes found it by running locally your patch 

```
### Processed [PHP] files
- Using [phpstan v1.7.15] https://megalinter.github.io/v5/descriptors/php_phpstan
- MegaLinter key: [PHP_PHPSTAN]
- Rules config: [.github/linters/phpstan.neon.dist]
--Error detail:
 ------ -----------------------------------------------------------------------
  Line   src/CallbackFilterHandler.php
 ------ -----------------------------------------------------------------------
  80     Parameter #1 $level of static method Monolog\Logger::toMonologLevel()
         expects
         100|200|250|300|400|500|550|600|'alert'|'ALERT'|'critical'|'CRITICAL'
         |'debug'|'DEBUG'|'EMERGENCY'|'emergency'|'error'|'ERROR'|'INFO'|'info
         '|'NOTICE'|'notice'|'WARNING'|'warning', int|string given.
 ------ -----------------------------------------------------------------------


 [ERROR] Found 1 error
```

Reply:
@mimmi20 I'd like you ignore only one line rather than disable PHPStan linter

See diff
```
diff --git a/.mega-linter.yml b/.mega-linter.yml
index 3a433a6..34196e7 100644
--- a/.mega-linter.yml
+++ b/.mega-linter.yml
@@ -14,7 +14,7 @@ DISABLE_LINTERS:
     - PHP_BUILTIN
     - PHP_PSALM
     # phpstan fails at the moment as it wants to run with PHP 8
-    - PHP_PHPSTAN
+    #- PHP_PHPSTAN
     - YAML_V8R
 EDITORCONFIG_EDITORCONFIG_CHECKER_CLI_LINT_MODE: project
 JSON_PRETTIER_FILTER_REGEX_EXCLUDE: "(composer.json)"
diff --git a/src/CallbackFilterHandler.php b/src/CallbackFilterHandler.php
index eab7fa6..53c3220 100644
--- a/src/CallbackFilterHandler.php
+++ b/src/CallbackFilterHandler.php
@@ -77,7 +77,7 @@ class CallbackFilterHandler extends AbstractHandler implements ProcessableHandle
             }
         }

-        $this->handlerLevel = Logger::toMonologLevel($level);
+        $this->handlerLevel = Logger::toMonologLevel($level); // @phpstan-ignore-line
         parent::__construct($level, $bubble);    // @phpstan-ignore-line

         $this->handler = $handler;
```

Reply:
I changed the lines